website/server/libs/taskManager.js

```diff
@@ -96,6 +96,7 @@ export async function createTasks (req, res, options = {}) {
       if (taskData.requiresApproval) {
         newTask.group.approval.required = true;
       }
+      newTask.group.sharedCompletion = taskData.sharedCompletion || 'individualCompletion';
```

Comment:
Ensure individualCompletion is an option. Also, let's move all these constants to our groupTasks Library.